### PR TITLE
fixed pay issues

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #, if available:* https://github.com/bcgov/entity/issues/7332

*Description of changes:*
- fixed pay logic around PAD and payment action mismatch

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: api/namex/models/payment.py [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
 
+from namex.constants import PaymentState
 from namex.models import State, db
 
 
 class Payment(db.Model):
 
     class PaymentActions(Enum):
         """Valid actions for a payment."""
-        COMPLETE='COMPLETE'
```
Comment (thorwolpert) [inline api/namex/models/payment.py:12]: this state is used by /services/namex-pay
i think CREATE was added, but maybe missed from this enum

**Review by thorwolpert — COMMENTED** [review]
Payment.PaymentActions.COMPLETE is used by /services/namex-pay, so that would need to get fixed too.
The move to CREATE looks good otherwise.

File: api/namex/models/payment.py (hunk 1 above)
Comment (kialj876) [reply api/namex/models/payment.py:12]: Yes I've included the namex-pay changes in this PR

Comment (thorwolpert) [reply api/namex/models/payment.py:12]: bah, missed the file path change. sweet.

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**